app/src/main/java/fr/free/nrw/commons/review/ReviewImageFragment.java

```diff
@@ -40,6 +42,9 @@
     @BindView(R.id.button_no)
     Button noButton;
 
+    private final String SAVED_USER = "saved_user";
```

Comment:
Could you please add a Javadoc comment explaining what the variable contains, with an example?